### DISCUSSION TITLE
Possible Improvements to Critical Cartography and Participatory Mapping

### DISCUSSION THREAD

**Discussion by jsxu315** [albertkun/21S-ASIAAM-191A#81] [Past Thinking Caps]
One barrier is lack of awareness; I probably would have considered Census and other “official” data to be a pretty good picture of the truth if I didn’t take classes like this one or hang out in the right circles. We would have to rethink the way we teach geography and social sciences in K-12; personally I had to unlearn the idea that political maps are absolute because that’s what I learned in school, and introducing kids to different types of maps at an early age could help make it so that there’s less to unlearn. I don’t know if someone’s already made a curriculum for this, but even if there was, it would probably take a long time and wading through a lot of red tape to actually be implemented in most classrooms.

Inviting guest speakers (if possible) and showing kids different types of maps, such as tactile maps, could be helpful. Off the top of my head, one exercise could be to let kids compare how they think of a place (like the classroom/school) to other students. For example, an athlete might have the sports fields, gym, and classroom as landmarks in their head, while another student might think of certain classrooms as their landmarks. An instructor could ask them questions such as if they think of certain maps as more/less correct and exploring why they see them that way. 

For participatory mapping, an ideal situation would be one where outsiders wouldn’t need to intervene. My first thought was a stopgap solution; distribute educational pamphlets/kits so that people can make maps without outsiders & have a repository where people can upload photos of maps and can take them down whenever they want. It would be even better if they could control who downloads the photo, but screenshots are still inevitable. Another flaw to this is that people might not have the time to do it if there’s a learning curve, and technological disparities can still be a barrier even if outsiders provide devices (lack of signal, etc.). And this solution still involves outsider intervention in the first step, so people in harder-to-access places might still be left behind, and there’s the problem that outsiders are still selecting certain things to include and exclude in the pamphlets. 

I feel like educational disparities are still one of the biggest barriers to more inclusive participatory mapping; addressing this involves slow structural changes. In the meantime, improving access to education where possible and requiring more rigorous ethical examinations could help make small changes.

**Comment by rh5140** [comment]
I agree education is probably the most effective way to improve awareness and that it unfortunately comes with too many other issues to consider first. I know this year, ethnic studies was voted to be added as a requirement for California high schools, but because of deadlines and the wide disparity in opinions, the resulting curriculum that gets passed might still be lacking. ([source](https://edsource.org/2021/a-final-vote-after-many-rewrites-for-californias-controversial-ethnic-studies-curriculum/651338))

I appreciate the thought you put into coming up with possible schemes for participatory mapping that limits outsider intervention and how you pointed out the possible shortcomings. This week's reading really opened my eyes to how many seemingly innocuous things still take power away from the community they're supposed to serve.

**Comment by brillianz** [comment]
This piece made me think a lot to the "unconventional" maps that I have come across as of late. While much of our ambitions as critical cartographers leads us to think about empowerment, access, and equity, what many of these maps reflect is an emphasis on power and capital. For instance, how many times have we used Google Maps to find a major bank chain in a city we're in? If we're looking for that trendy new kombucha drink, how many times have we hopped on their website to search for the closest chain store that sells it? 

Unfortunately, the emphasis on capital for growth in mapping has not yet tapped into social justice issues with the same emphasis. It falls on the ambitions of independent, critical cartographers to establish accessible maps that can connect people to resources in an accessible way. I can think of bus route maps as one of the only institutional examples of maps that truly help people out. I look forward to doing my best to establish more of those as I learn more from this class.
